app-text/sword/sword-1.8.1.recipe

```diff
@@ -6,18 +6,21 @@ societies to write new Bible software more quickly and easily. We also create \
 Bible study software for all readers, students, scholars, and translators of \
 the Bible, and have a growing collection of over 200 texts in over 50 languages."
 HOMEPAGE="http://www.crosswire.org/sword"
-COPYRIGHT="1998-2014 CrossWire Bible Society"
+COPYRIGHT="1998-2018 CrossWire Bible Society"
 LICENSE="GNU GPL v2"
-REVISION="3"
-SOURCE_URI="http://www.crosswire.org/ftpmirror/pub/sword/source/v1.7/sword-1.7.3.tar.gz"
-CHECKSUM_SHA256="5a3d87ff155d5ecb0cfec052ba333b2b74d9273e2cc66fb1ca75747dfd8ea9ea"
+REVISION="1"
+SOURCE_URI="http://www.crosswire.org/ftpmirror/pub/sword/source/v1.8/sword-$portVersion.tar.gz"
```

Comment:
Suggestion:
```diff
-SOURCE_URI="http://www.crosswire.org/ftpmirror/pub/sword/source/v1.8/sword-$portVersion.tar.gz"
+SOURCE_URI="http://www.crosswire.org/ftpmirror/pub/sword/source/v${portVersion%.*}/sword-$portVersion.tar.gz"
```
(This is safe because all versions seem to follow the `x.y.z` pattern.)